cg/scripts/blockchain-function-test.js

```diff
@@ -0,0 +1,32 @@
+const {
```

Comment:
This test script is good to test the new smart contract functions inside the system, so let's leave it here now with a nicer name ;)